### PR TITLE
Create your own Enumerable

### PR DESCRIPTION
**Pull request by usmansbk**
# Create your own Enumerable

In this PR, we did the following:

- Create a class MyList that has an instance variable @list.
- In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.
- Create a module MyEnumerable that implements the following methods:
  -  #all?
  -  #any?
  -  #filter

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+end
```
Comment (DeliceLydia) [inline my_list.rb]: - [x] The `initialize` method should accept multiple arguments let us say if I do something like `MyList.new(1, 2, 3, 4)` calling the constructor with this list of numbers will be possible kindly work on that

Comment (DeliceLydia) [reply my_list.rb]: - [x] Please implement `each` method that yields all members of @list in succession remember almost every method depends on `each` method to work

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb (hunk 1 above)
Comment (usmansbk) [reply my_list.rb]: @DeliceLydia Intialize does work on multiple arguments. That's a varargs

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**